src/lib/mina_lib/mina_lib.ml

```diff
@@ -1299,6 +1300,9 @@ let create ?wallets (config : Config.t) =
                       ~conf_dir:config.conf_dir))
             >>| Result.ok_exn
           in
+          [%log' info config.logger] "Took %f ms to create prover process"
```

Comment:
This logs probably want to be at a lower level than `info`.

Reply:
Nit: use metadata to pass this info, so that we can search for `Took $time ms to create prover process` in stackdriver.